### ISSUE TITLE
Image separated from footer on large screens

### ISSUE THREAD

**Issue by nclm** [mastodon/joinmastodon#210]
Great new website!
Tiny bug on big screens: on viewports that are larger than 2560px, the image above the footer gets separated from the footer with a white space:
![screenshot](https://user-images.githubusercontent.com/923008/191912345-3100c908-23ee-4752-98ac-1789bee1ce24.png)

**Comment by ChaosExAnima** [comment]
should be fixed now, closing.
